packages/calypso-e2e/src/lib/blocks/file-block.ts

```diff
@@ -40,7 +40,7 @@ export class FileBlock {
 		// It can lead to the filename placeholder text not being replaced with the uploaded file name.
 		await Promise.all( [
 			page.waitForResponse(
-				( response: Response ) => response.url().includes( 'media?' ) && response.status() === 200
+				( response: Response ) => response.url().includes( 'media?' ) && response.ok()
```

Comment:
**question/non-blocking**: was the check for 200 causing unexpected failures?

Reply:
Yep! The actual issue was that the Simple sites are responding with 200 and Atomic with 201 (resource created). This would probably be good to have unified I guess. What do you think?